app/components/post_list/date_header/date_header.js

```diff
@@ -31,18 +32,25 @@ export default class DateHeader extends PureComponent {
         } else {
             date = new Date(parseInt(dateLineString.substring(DATE_LINE.length), 10));
         }
+        const dateFormatProps = {
+            weekday: 'short',
+            day: '2-digit',
+            month: 'short',
+            year: 'numeric',
+            value: date,
+        };
+
+        if (timeZone) {
+            dateFormatProps.timeZone = timeZone;
```

Comment:
Ugh, I hate that this component takes `timeZone` instead of `timezone`. That definitely causes a ton of typos.